projects/observability/src/shared/graphql/request/builders/specification/entity/secondary-entity-specification-builder.ts

```diff
@@ -0,0 +1,54 @@
+import { Dictionary } from '@hypertrace/common';
+import { GraphQlArgumentBuilder } from '@hypertrace/distributed-tracing';
+import { GraphQlSelection } from '@hypertrace/graphql-client';
+import { Entity, entityIdKey, EntityType, entityTypeKey } from '../../../../model/schema/entity';
+import { EntitySpecification } from '../../../../model/schema/specifications/entity-specification';
+
+export class SecondaryEntitySpecificationBuilder {
```

Comment:
Can we reuse existing ObservabilitySpec builder for this?

Reply:
Yes!